bitnami/keycloak/templates/tls-secret.yaml

```
@@ -9,16 +9,16 @@ SPDX-License-Identifier: APACHE-2.0
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ .name }}
+  name: {{ tpl .name $ }}
```

Could you please use our own function for this instead of directly using `tpl`? It is used further below in L16:

```suggestion
  name: {{- include "common.tplvalues.render" ( dict "value" .name "context" . ) }}
```

---

Yes, sure, I could do that. You can see the usage of both throughout the chart. I could not fully understand why isn't always "common.tplValues.render" used. My only guess was simpler cases are used with tpl and more complex ones with  "common.tplValues.render". This looked simpler for me.

---

FYI, this other guy is me, from the wrong account.

---

> I could not fully understand why isn't always "common.tplValues.render" used

`tpl` is the legacy approach used in our charts and `common.tplValues.render` the new one. In newer charts (and features) you will exclusively see the latter being used 😁